src/tests/epicTargetingTestData.ts

```diff
@@ -0,0 +1,166 @@
+    ...SHARED_PARAGRAPHS,
+];
+
+const EU_ROW_LESS_ENGAGED_PARAGRAPHS = [
+    '... as you join us today from %%COUNTRY_NAME%%, we have a small favour to ask. Through these turbulent and challenging times, millions rely on the Guardian for independent journalism that stands for truth and integrity. Readers chose to support us financially more than 1.5 million times in 2020, joining existing supporters in 180 countries.',
+    ...SHARED_PARAGRAPHS,
+];
+
+// ---- Other variant fields ---- //
+
+const HIGHLIGHTED_TEXT =
+    'Support the Guardian from as little as %%CURRENCY_SYMBOL%%1 – it only takes a minute. If you can, please consider supporting us with a regular amount each month. Thank you.';
+
+const CTA: Cta = {
+    text: 'Support The Guardian',
+    baseUrl: 'https://support.theguardian.com/contribute',
+};
+
+// ---- Variants ---- //
+
+const BASE_VARIANT: Variant = {
+    name: '',
+    modulePathBuilder: epic.endpointPathBuilder,
+    paragraphs: [],
+    highlightedText: HIGHLIGHTED_TEXT,
+    cta: CTA,
+};
+
+function getVariants(paragraphs: string[], kind: 'HIGHLY_ENGAGED' | 'LESS_ENGAGED'): Variant[] {
+    const control = {
+        ...BASE_VARIANT,
+        name: 'control',
+        paragraphs,
+        maxViews: CONTROL_MAX_VIEWS,
+    };
+
+    const variant =
+        kind === 'HIGHLY_ENGAGED'
+            ? {
+                  ...BASE_VARIANT,
+                  name: 'ask6',
+                  paragraphs,
+                  maxViews: ASK6_MAX_VIEWS,
+              }
+            : {
+                  ...BASE_VARIANT,
+                  name: 'ask2',
+                  paragraphs,
+                  maxViews: ASK2_MAX_VIEWS,
+              };
+
+    return [control, variant];
+}
+
+export const UK_AUS_HIGHLY_ENGAGED_VARIANTS = getVariants(
+    UK_AUS_HIGHLY_ENGAGED_PARAGRAPHS,
+    'HIGHLY_ENGAGED',
+);
+export const UK_AUS_LESS_ENGAGED_VARIANTS = getVariants(
+    UK_AUS_LESS_ENGAGED_PARAGRAPHS,
+    'LESS_ENGAGED',
+);
+
+export const US_HIGHLY_ENGAGED_VARIANTS = getVariants(
+    US_HIGHLY_ENGAGED_PARAGRAPHS,
+    'HIGHLY_ENGAGED',
+);
+export const US_LESS_ENGAGED_VARIANTS = getVariants(US_LESS_ENGAGED_PARAGRAPHS, 'LESS_ENGAGED');
+
+export const EU_ROW_HIGHLY_ENGAGED_VARIANTS = getVariants(
+    EU_ROW_HIGHLY_ENGAGED_PARAGRAPHS,
+    'HIGHLY_ENGAGED',
+);
+export const EU_ROW_LESS_ENGAGED_VARIANTS = getVariants(
+    EU_ROW_LESS_ENGAGED_PARAGRAPHS,
+    'LESS_ENGAGED',
+);
+
+// ---- Articles viewed settings ---- //
+
+export const HIGHLY_ENGAGED_ARTICLES_VIEWED_SETTINGS: ArticlesViewedSettings = {
+    minViews: 5,
+    periodInWeeks: 52,
+};
+
+export const LESS_ENGAGED_ARTICLES_VIEWED_SETTINGS: ArticlesViewedSettings = {
+    minViews: 0,
+    maxViews: 5,
+    periodInWeeks: 52,
+};
+
+// ---- Tests ---- //
+
+export const BASE_TEST: Test = {
+    name: '',
+    campaignId: '',
```

Comment:
shoud this be set to something meaningful?

Reply:
again, I would leave off the `Test` type and just define the common fields here. Let the typechecker ensure a full `Test` object is created elsewhere

Reply:
oh I see - you're talking specifically about campaignId!
It's typically `testName_variantName`

Reply:
An alternative approach is to make it a function:
`const buildTest = (name: string, locations...): Test => ({...})`

Reply:
Also, I got `campaignId` confused with `campaignCode` - I think it can just be the name of the test. I'm also not sure it's even used!

Reply:
It wasn't quite as easy as just dropping the `Test` type, but I went a little exotic and used the `Omit` type to remove the fields we want each test to implement - what do you think?

Reply:
(the reason is `userCohort` is just inferred as type string which TS isn't happy with)

Reply:
Ah I see. The alternative is to make it a function